pkg/core/blockchainer/policer.go

```diff
@@ -0,0 +1,17 @@
+package blockchainer
+
+import (
+	"math/big"
+
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// Policer is an interface that abstracts the implementation of policy methods.
+type Policer interface {
+	GetDepositBalance(acc util.Uint160) *big.Int
+	GetDepositExpiration(acc util.Uint160) uint32
```

Comment:
This isn't part of the policer, it's something different.

Reply:
Moved out of the Policer to the Blockchainer.